stripe/src/test/java/com/stripe/android/StripeKotlinTest.kt

```diff
@@ -0,0 +1,545 @@
+package com.stripe.android
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import com.stripe.android.exception.APIException
+import com.stripe.android.exception.AuthenticationException
+import com.stripe.android.exception.InvalidRequestException
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.SetupIntent
+import com.stripe.android.model.Source
+import com.stripe.android.networking.ApiRequest
+import com.stripe.android.networking.StripeApiRepository
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import kotlin.test.assertFailsWith
+import kotlin.test.assertSame
+
+/**
+ * Test for [Stripe] suspend functions.
+ */
+@RunWith(JUnit4::class)
+internal class StripeKotlinTest {
+    private val mockApiRepository: StripeApiRepository = mock()
+    private val mockPaymentController: PaymentController = mock()
+
+    private val stripe: Stripe =
+        Stripe(mockApiRepository, mockPaymentController, TEST_PUBLISHABLE_KEY)
+
+    @Test
+    fun `When repository returns correct value then createPaymentMethod should Succeed`(): Unit =
```

Comment:
You should be able to drop `Unit` return type

Reply:
looks like JUnit needs a test retuning null explicitly, got this error if `Unit` is removed

> Method When repository throws exception then createPaymentMethod should throw same exception() should be void